File: app/src/main/java/com/chess/cryptobot/view/SettingsFragment.java
package com.chess.cryptobot.view;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import com.chess.cryptobot.R;

public class SettingsFragment extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.preferences, rootKey);
    }
}

File: app/src/main/java/com/chess/cryptobot/view/SettingsFragment.kt
package com.chess.cryptobot.view

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.chess.cryptobot.R

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.preferences, rootKey)
    }
}